planet/src/main/java/com/google/planet/data/TimeRange.java

```diff
@@ -64,8 +64,8 @@ private static boolean equals(TimeRange a, TimeRange b) {
     }
 
     public static int getTimeInMinutes(int hours, int minutes) {
-        if (hours < 8 || hours > 19) {
-            throw new IllegalArgumentException("Hours can only be 8 through 19 (inclusive).");
+        if (hours < 0 || hours > 23) {
```

Comment:
These changes are not relevant to the PR (get place details from the google maps place details api). You can separate changes like these in another PR. It will land faster and offers more clarity to whoever looks at the commit history in the future.

Reply:
Okay!